### PR TITLE
101130644 packer slave

### PR DESCRIPTION
**Pull request by johnjmaguire**
Jenkins Slave with Packer and Chef installed for building AMIs. Currently in use on Jenkins-Devops for our Base AMI. Magic!

### CODE REVIEW

File: packer-slave/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM shopkeep/jenkins-devops-slave
+MAINTAINER John Maguire <jmaguire@shopkeep.com>
+
+RUN apt-get update && apt-get install -y \
```
Comment (pguelpa) [inline packer-slave/Dockerfile]: Not a change for now, but I've been wondering about this for a while. Do you think it's worthwhile for us to make a ShopKeep base image and have it do the `apt-get update` and tag that with a date? I always wonder about it when I write it out as I'm not sure what version of the thing I'm getting updated to.

**Comment by pguelpa** [comment]
:+1:

**Comment by pguelpa** [comment]
Oh, could we toss a README.md in here too? Just to give us a place to put any information about how to use this container if we need it.

File: packer-slave/Dockerfile (hunk 1 above)
Comment (johnjmaguire) [inline packer-slave/Dockerfile]: You know I actually had something like this in place when I built that shopkeep-ruby Dockerfile. Originally it didn't come directly FROM ubuntu, it came from another image called shopkeep-base which in turn came FROM ubuntu and I added the deps there. I still think it's a pretty valid setup I just wasn't sure how many 'chained' images was too many!

Comment (pguelpa) [inline packer-slave/Dockerfile]: Yeah, I'm not really sure on that either. I think the biggest problem Docker is is with too many layers. I don't really know enough about how often apt-get repos get updated to know what the best practice would be, but it seems like a nice thing to do to ensure reproducible builds.

**Comment by paddycarey** [comment]
@johnjmaguire @pguelpa we still planning to merge this PR or can it be closed?

**Comment by pguelpa** [comment]
I'm not sure, I'll defer to @johnjmaguire

**Pull request closed without merging**